app/middleware/catch_json_parse_errors.rb

```diff
@@ -8,7 +8,7 @@ def call(env)
     @app.call(env)
   rescue ActionDispatch::ParamsParser::ParseError => error
     if env['HTTP_ACCEPT'] =~ /application\/json/
-      error_output = "There was a problem in the JSON you submitted: #{error}"
+      error_output = "Invalid JSON"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.